main.py

```diff
@@ -86,16 +89,20 @@ def reinforce(env, model, episode, total_reward, stddev=1.0):
     gt = total_reward
     var = stddev ** 2
     for state_rep, pred, action, reward in episode:
-        action_t = action.transpose()
+        action_t = np.array(action).transpose()
         target = (action_t - pred)/var * gt + pred
         model.train_on_batch(state_rep, target)
         gt -= reward
     
 def create_model(k):
     model = Sequential()
-    model.add(Dense(units=512, input_dim = k * 4 + 4))
+    model.add(Dense(units=56, input_dim = k * 4 + 4))
     model.add(Activation('relu'))
-    model.add(Dense(units=512))
+    model.add(Dense(units=56))
+    model.add(Activation('relu'))
+    model.add(Dense(units=56))
+    model.add(Activation('relu'))
+    model.add(Dense(units=56))
```

Comment:
no need for change but I think adding more layers to the network won't make it easier to debug since the function approximator actually becomes slower to train (vanishing gradients problem). I think it might be better to test by reducing complexity.